frontend/src/components/header/mobileLogo/MobileLogo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Typography} from "@mui/material";
+import './styles.css';
+import {Titles} from "../../../common/constansts/titles";
+
+const MobileLogo: React.FC = () => {
+    return (
+        <Typography
+            variant="h5"
+            noWrap
+            component="a"
```

Comment:
Is "a" really what you want ? I'm just asking, I don't know

Reply:
no, it isn't necessary, forgot to delete it